src/logger.ts

```diff
@@ -78,11 +78,18 @@ function log(
   }
 }
 
-const blockStore: Block = ['store', '%s', styles.store];
 const blockNew: Block = ['new', '%s', styles.new];
+
+const blockStore: Block = ['store', '%s', styles.store];
 const blockEvent: Block = ['event', '%s', styles.event];
 const blockEffect: Block = ['effect', '%s', styles.effect];
 
+const stripDomain = (name: string) => name.split('/').pop() || name;
```

Comment:
Why do you want to strip the domain name from the log?

Reply:
This is only necessary for the first block, to replace the repeated words event store effect, since the full name including the domain can be too long, however, to preserve the ability to filter by the full name or domain, the full name is displayed at the end of the log.
The color differentiation of various units is also preserved.